Services/Export/classes/class.ilExport.php

```diff
@@ -602,6 +603,10 @@ public function processExporter(
         $this->log->debug("process tail dependencies of " . $a_entity);
         $sequence = $exp->getXmlExportTailDependencies($a_entity, $a_target_release, $a_id);
         foreach ($sequence as $s) {
+            if (empty((array) $s["ids"])) {
```

Comment:
Please make this more robust against "undefined array key" failures.

Reply:
@smeyer-ilias Wish I could take credit but I think you're looking for someone else 😇

Reply:
Sorry and thanks for your reply

Reply:
Done, thanks @smeyer-ilias (and also to our intercontinental consultant)